fastcrypto-zkp/src/api.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
i created a new mod here to isolate the dependencies. happy to put this inside verifier.rs as well